config/database.yml

```diff
@@ -82,5 +86,7 @@ test:
 production:
   <<: *default
   database: rails_recipe_app_production
-  username: rails_recipe_app
-  password: <%= ENV["RAILS_RECIPE_APP_DATABASE_PASSWORD"] %>
+  # username: rails_recipe_app
+  # password: <%= ENV["RAILS_RECIPE_APP_DATABASE_PASSWORD"] %>
+  username: postgres
+  password: pw4Admin
```

Comment:
It can be better if we keep the databases config file without any username and password because those are going to be different from one computer to another.